documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,83 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Advantages and disadvantages of cold and warm migrations
+
+[id="warm-cold-migration-overview_{context}"]
+=== Overview
+
+Both cold migration and warm migration have advantages and disadvantages, as described in the table that follows:
+
+[cols="1,1,1",options="header"]
+.Advantages and disadvantages of cold and warm migrations
+
+|===
+| |Duration |VM downtime
+
+|Cold migration
+|Minimal
+|High
+
+|Warm migration
+|Longer and variable
+|Minimal
+|===
+
```

Comment:
@RichardHoch is there any way you could convert this to a snippet? WDYT

i might want to use it in the release notes

Reply:
For this place I'd keep a table, as it's easier to parse IMO

Reply:
@fabiand He's asking me to create a document snippet containing the table, which he will then use in the release notes -- if I can do that, it will look the same to readers. if I can't, I'll leave the file as is. No worries!